tasks/main.yml

```diff
@@ -3,6 +3,60 @@
 - name: Pre hooks
   include: '{{ lookup("task_src", "sshd/pre_main.yml") }}'
 
+- name: Check if OpenSSH is installed (or half-installed)
+  script: scripts/dpkg-state openssh-server
+  register: sshd__prev_pkg_state
```

Comment:
This variable should be named `sshd__register_prev_pkg_state` to keep the correct variable namespace.